From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Thu, 13 Jun 2019 10:50:16 -0700
Subject: [PATCH] Relocate debuginfo to /usr/share/debug

For future use as a lookaside debuginfo tree, package the binary
debuginfo in /usr/share/debug and the source debuginfo in
/usr/share/debug/src.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 scripts/find-debuginfo.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/scripts/find-debuginfo.sh b/scripts/find-debuginfo.sh
index 779a9ff..87e9511 100755
--- a/scripts/find-debuginfo.sh
+++ b/scripts/find-debuginfo.sh
@@ -674,3 +674,5 @@ if ((nout > 0)); then
   cat "$LISTFILE" >> "${LISTFILE}.new"
   mv "${LISTFILE}.new" "$LISTFILE"
 fi
+sed -i -e "s|/usr/lib/debug|/usr/share/debug|g" $LISTFILE
+sed -i -e "s|/usr/src/debug|/usr/share/debug/src|g" $LISTFILE
